newIDE/app/src/CompactPropertiesEditor/index.js

```diff
@@ -0,0 +1,791 @@
+        //     multiline
+        //     style={styles.field}
+        //   />
+        // );
+      } else if (field.valueType === 'resource') {
+        return 'TODO';
+      } else {
+        const {
+          // TODO: Still support onEditButtonClick & onEditButtonBuildMenuTemplate ?
+          // onEditButtonBuildMenuTemplate,
+          // onEditButtonClick,
+          setValue,
+          onClickEndAdornment,
+        } = field;
+        return (
+          <CompactSemiControlledTextField
+            value={getFieldValue({
+              instances,
+              field,
+              defaultValue: '(Multiple values)',
+            })}
+            id={field.name}
+            // floatingLabelText={getFieldLabel({ instances, field })}
+            // helperMarkdownText={getFieldDescription(field)}
+            onChange={newValue => {
+              instances.forEach(i => setValue(i, newValue || ''));
+              _onInstancesModified(instances);
+            }}
+            disabled={getDisabled({ instances, field })}
+            renderLeftIcon={field.renderLeftIcon}
+            leftIconTooltip={getFieldLabel({ instances, field })}
+            renderEndAdornmentOnHover={field.getEndAdornmentIcon || undefined}
+            onClickEndAdornment={() => {
+              if (!onClickEndAdornment) return;
+              instances.forEach(i => onClickEndAdornment(i));
+              _onInstancesModified(instances);
+            }}
+          />
+        );
+      }
+    },
+    [instances, _onInstancesModified]
+  );
+
+  const renderSelectField = React.useCallback(
+    (field: ValueField) => {
+      if (!field.getChoices || !field.getValue) return;
+
+      const children = field
+        .getChoices()
+        .map(({ value, label, labelIsUserDefined }) => (
+          <SelectOption
+            key={value}
+            value={value}
+            label={label}
+            shouldNotTranslate={labelIsUserDefined}
+          />
+        ));
+
+      if (field.valueType === 'number') {
+        const { setValue } = field;
+        return (
+          <CompactSelectField
+            value={getFieldValue({ instances, field })}
+            key={field.name}
+            id={field.name}
+            // floatingLabelText={getFieldLabel({ instances, field })}
+            // helperMarkdownText={getFieldDescription(field)}
+            onChange={(newValue: string) => {
+              instances.forEach(i => setValue(i, parseFloat(newValue) || 0));
+              _onInstancesModified(instances);
+            }}
+            disabled={field.disabled}
+          >
+            {children}
+          </CompactSelectField>
+        );
+      } else if (field.valueType === 'string') {
+        const { setValue } = field;
+        return (
+          <CompactSelectField
+            value={getFieldValue({
+              instances,
+              field,
+              defaultValue: '(Multiple values)',
```

Comment:
I don't remember seeing this in the app, should this be translated?

Reply:
It's already in the app, I reused it as is, but I will translate it indeed.